Revert-and-Regression-Policy.md

```diff
@@ -1,14 +1,22 @@
 ## Table of Contents
 
+* [Blame-Free Reverts](#blame-free-reverts)
 * [Policy](#policy)
 * [Instructions](#instructions)
   * [Developers Who Notice a Regression](#developers-who-notice-a-regression)
   * [Developers Who Cause a Regression](#developers-who-cause-a-regression)
+  * [Developers with Revert Permissions](#developers-with-revert-permissions)
   * [Dev Workflow Leads](#dev-workflow-leads)
 * [Tips](#tips)
   * [How to identify the bad commit](#how-to-identify-the-bad-commit)
   * [How to revert a PR](#how-to-revert-a-pr)
 
+## Blame-Free Reverts
+
+In the past, reverts have sometimes been seen as a punitive measure . This is no longer the case at Oppia. Often, PRs need to be reverted despite the PR author doing nothing wrong. For example, sometimes a race condition occurs where 2 PRs are merged in quick succession and conflict. We may have to revert one of the PRs to fix develop even though no PR author did anything wrong.
```

Comment:
I'm a bit unsure about the first two sentences. I don't think we've ever intentionally applied reverts punitively, though the perception to the revertee may be that reverts are punitive. Both that policy (of not intentionally applying reverts punitively) and perception may well still continue into the future, so I don't think that something has necessarily changed there. The main thing that's changed is that we now have a wiki page that folks can reference and that puts more structure around the process.

So, perhaps start this paragraph with "Sometimes, PRs need to be reverted because of XXX." and omit the first two sentences, then have the second paragraph start with "It is important to note that reverts are not meant to be punitive. A key tenet ..."?

Reply:
fixed